supabase/migrations/20240425205531_create_auth_challege_table.sql

```diff
@@ -0,0 +1,36 @@
+create table "public"."challenges" (
+    "id" serial primary key,
+    "challenge" bytea not null,
```

Comment:
sorry, i don't know why I didn't suggest this earlier, but it probably would have been much easier to just use `pg_crypto.gen_random_bytes`


```suggestion
    "challenge" bytea not null default gen_random_bytes(64),
```

https://www.postgresql.org/docs/current/pgcrypto.html#PGCRYPTO-RANDOM-DATA-FUNCS

Reply:
it's fine to leave as is too. just informational

Reply:
refactored. thanks! much cleaner this way.